lib/ruby_lsp/setup_bundler.rb

```diff
@@ -280,5 +281,15 @@ def correct_relative_remote_paths
 
       @custom_lockfile.write(content)
     end
+
+    # Detects if the project is a Rails app by looking if the superclass of the main class is `Rails::Application`
+    sig { returns(T::Boolean) }
+    def rails_app?
+      config = Pathname.new("config/application.rb").expand_path(Dir.pwd)
+      application_contents = config.read if config.exist?
+      return false unless application_contents
+
+      /class .* < Rails::Application/.match?(application_contents)
```

Comment:
Should we proof this a bit more? I know the file is generated but I wonder if someone would have some weird cop doing this kind of change?

```suggestion
      /class .* < (::)?Rails::Application/.match?(application_contents)
```

Reply:
This makes sense, I've committed it.